cwms-data-api/src/test/java/cwms/cda/data/dto/timeseriesprofile/TimeSeriesProfileParserTest.java

```diff
@@ -21,6 +21,7 @@ void testTimeSeriesProfileColumnarSerializationRoundTrip() {
         ContentType contentType = Formats.parseHeader(Formats.JSONV1, TimeSeriesProfileParserColumnar.class);
 
         String serialized = Formats.format(contentType, timeSeriesProfileParser);
+        assert(serialized.contains("type"));
```

Comment:
use `assertTrue` instead of all these other vanilla asserts

Reply:
Replaced with `assertTrue`